src/index.html

```diff
@@ -1,11 +1,138 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Homework 1</title>
+    <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1>Start developing here</h1>
+    <header class="header">
+      <div>
+        <a href="#" type="button">Web<b>Studio</b></a>
+        <nav>
+          <ul>
+            <li class="1"><a href="#">Студія</a></li>
+            <li class="b"><a href="#">Портфоліо</a></li>
+            <li class="c"><a href="#">Контакти</a></li>
+          </ul>
+        </nav>
+        <a href="#">info@devstudio.com</a>
+        <p>+38 096 111 11 11</p>
+      </div>
+    </header>
+    <main>
+      <div>
+        <h1>Ефективні рішення для вашого бізнесу</h1>
+        <button name="button">Замовити послугу</button>
+      </div>
+      <section>
+        <ul>
+          <li><b>УВАГА ДО ДЕТАЛЕЙ</b></li>
```

Comment:
Should be like this:
<ul>
  <li>
     <h2>...</h2>
     <p>...</p>
  </li>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ul>